engine/HAL/STM32L0/luos_hal.c

```diff
@@ -0,0 +1,328 @@
+/******************************************************************************
+ * @file luosHAL
+ * @brief Luos Hardware Abstration Layer. Describe Low layer fonction
+ * @MCU Family STM32L0
+ * @author Luos
```

Comment:
I think you commit comment `add HAL for L0 familly` should be `Add HAL for STM32L0 familly`